src/vs/workbench/contrib/chat/browser/actions/chatAccessibilityHelp.ts

```diff
@@ -22,3 +54,40 @@ function descriptionForCommand(commandId: string, msg: string, noKbMsg: string,
 	}
 	return format(noKbMsg, commandId);
 }
+
+export async function runAccessibilityHelpAction(accessor: ServicesAccessor, editor: ICodeEditor, type: 'chat' | 'editor'): Promise<void> {
+	const widgetService = accessor.get(IChatWidgetService);
+	const inputEditor: ICodeEditor | undefined = type === 'chat' ? widgetService.lastFocusedWidget?.inputEditor : editor;
+	const editorUri = editor.getModel()?.uri;
+
+	if (!inputEditor || !editorUri) {
+		return;
+	}
+	const domNode = withNullAsUndefined(inputEditor.getDomNode());
+	if (!domNode) {
+		return;
+	}
+
+	const cachedInput = inputEditor.getValue();
+	const cachedPosition = inputEditor.getPosition();
+	inputEditor.getSupportedActions();
+	const helpText = getAccessibilityHelpText(accessor, type, type === 'editor' ? cachedInput : undefined);
+	inputEditor.setValue(helpText);
+	inputEditor.updateOptions({ readOnly: true });
+	inputEditor.focus();
+	const disposable = addStandardDisposableListener(domNode, 'keydown', e => {
```

Comment:
Nervous about this, I wonder if it also needs to be disposed when the editor closes or the chat widget goes away.